data/init-mongo.js

```diff
@@ -1,4 +1,18 @@
 db = db.getSiblingDB("stamfordcenter");
+
+db.announcements.insertMany([
+	{
+		_id: "65212e74aac38c41507570b3",
+		subject: "STIU Market",
+		message: "Get ready for upcoming STIU Market on Thursday!",
+	},
+	{
+		_id: "65212e74aac38c21507570f8",
+		subject: "Final Exams",
+		message: "This is Week12 of the semester! Everyone ready for final exams?",
+	},
+]);
+
```

Comment:
Heyy, can you adjust it to follow this format?

![image](https://github.com/stamford-syntax-club/stamfordcenter-backend/assets/92321280/3385bab3-4db4-4d20-a2db-05cd3747f9e1)